openforcefield/tests/test_parameters.py

```diff
@@ -1691,14 +1691,27 @@ def test_charge_increment_type_wrong_num_increments(self):
                 charge_increment3=-0.1 * unit.elementary_charge,
             )
 
-        with pytest.raises(
-            SMIRNOFFSpecError,
-            match="initialized with unequal number of tagged atoms and charge increments",
-        ) as excinfo:
-            ci_type = ChargeIncrementModelHandler.ChargeIncrementType(
-                smirks="[#6:1]-[#7:2]-[#6]",
-                charge_increment1=0.05 * unit.elementary_charge,
-            )
+        ci_type = ChargeIncrementModelHandler.ChargeIncrementType(
+            smirks="[#6:1]-[#7:2]-[#6]",
+            charge_increment1=0.05 * unit.elementary_charge,
+        )
+
+    def test_charge_increment_infer_one(self):
+        """Test the simplest case of automatically inferring a missing value"""
+        inferred = ChargeIncrementModelHandler.ChargeIncrementType(
+            smirks="[*:1]-[*:2]",
+            charge_increment=[0.1 * unit.elementary_charge],
+        )
+
+        explicit = ChargeIncrementModelHandler.ChargeIncrementType(
+            smirks="[*:1]-[*:2]",
+            charge_increment=[
+                0.1 * unit.elementary_charge,
+                -0.1 * unit.elementary_charge,
+            ],
+        )
+
+        assert inferred.to_dict() == explicit.to_dict()
```

Comment:
I'm removing the equality comparison at the end of this test, since I think that a chargeincrement with one unspecified value shouldn't automatically add that value during serialization, or make that value accessible before parameterization.